account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/UpdateEmailHandler.java

```diff
@@ -149,6 +147,36 @@ public APIGatewayProxyResponseEvent updateEmailRequestHandler(
                                             new UserNotFoundException(
                                                     "User not found with given email"));
 
+            AtomicReference<EmailCheckResultStatus> resultStatus =
+                    new AtomicReference<>(EmailCheckResultStatus.PENDING);
+            dynamoEmailCheckResultService
+                    .getEmailCheckStore(updateInfoRequest.getReplacementEmailAddress())
+                    .ifPresent(result -> resultStatus.set(result.getStatus()));
+            LOG.info(
+                    "UpdateEmailHandler: Experian email verification status: {}",
+                    resultStatus.get());
+            if (resultStatus.get().equals(EmailCheckResultStatus.PENDING)) {
+                auditService.submitAuditEvent(
```

Comment:
Can these audit event calls use the `submitAuditEvent(AuditableEvent event, AuditContext auditContext)` signature instead?